### PR TITLE
Verbose flag

### PR DESCRIPTION
**Pull request by copelco**
Support increasing verbosity of underlying ansible-playbook command, e.g. ``deploy:verbose=vv``

### CODE REVIEW

File: tequila_fab/__init__.py [hunk 1]
```diff
@@ -74,6 +74,8 @@ def deploy(play=None, extra_vars=None, branch=None, limit=None):
         cmd.append("-e repo_branch=%s" % branch)
     if limit:
         cmd.append("-l %s" % limit)
+    if verbose:
+        cmd.append("-%s" % verbose)
```
Comment (vkurup) [inline tequila_fab/__init__.py:82]: I'm wondering if we should do a little validation to make sure that `verbose` is a string containing only the letter `v` (`set(verbose) == {'v'}`). If it doesn't, I'm not sure what the better option is.... 
1) ignoring the parameter, or
2) setting verbose to `vvv`, assuming that the user wants more verbosity

**Review by vkurup — APPROVED** [review]
Looks like a useful change! I made one comment, but I don't feel too strongly about it. Would be nice to add a line in the README about this addition to the deploy command.

File: tequila_fab/__init__.py (hunk 1 above)
Comment (copelco) [reply tequila_fab/__init__.py:82]: Good idea, added!

**Review by vkurup — APPROVED** [review]
Nice solution!

**Pull request merged**